source/administration/upgrade.md

```diff
@@ -130,6 +130,7 @@ If your Mattermost server has duplicate accounts (users with multiple accounts i
 5. Reset the Mattermost account as the directory owner by typing:
  - `sudo chown -R mattermost:mattermost <path-to-your-mattermost-folder>`
  - `sudo chmod -R g+w <path-to-your-mattermost-folder>`
+6. To force the client to refresh for your users, save a setting in the System Console. For example, go to **System Console** > **General** > **Configuration**, and copy the Listen Address. Then paste the Listen Address back into the field, and press **Save**.
```

Comment:
Propose `To force the client to refresh for your users...` > `To ensure clients applications don't encounter issues with out-of-sync caches, you can force clients to refresh after a server upgrade by changing any setting in the System Console.`

Also, is there a ticket to improve this functionality in future so the forced refresh automatically happens after a server upgrade?